test/system/src/test/java/io/pravega/test/system/MetadataScalabilityTest.java

```diff
@@ -47,24 +46,14 @@
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
 
-/**
- * This test creates a stream with 10k segments and then rapidly scales it 1010 times.
- * Then it performs truncation a random number (less than 1010) of times. 
- */
 @Slf4j
```

Comment:
Adding some comments to describe the general purpose of this abstract class would be great.